app/views/videos/show.html.haml

```diff
@@ -10,4 +10,33 @@
         %p #{@video.description}
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
\ No newline at end of file
+          = link_to "+ My Queue", queue_items_path(video_id: @video.id), method: :post, class: 'btn btn-default'
+          
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([5,4,3,2,1].map { |number| [pluralize(number, "Star")]})
```

Comment:
I would recommend moving this part `[5,4,3,2,1].map { |number| [pluralize(number, "Star")]}` to a helper. It makes it easier to read and you will need that logic later on anyway.